src/config/index.ts

```diff
@@ -493,3 +493,5 @@ export const PAYMENT_WITH_TRANSACTION_WITH_PARAMS_URL = REGISTER_AN_OVERSEAS_ENT
 // Trusts with id params
 export const TRUST_ENTRY_WITH_PARAMS_URL = REGISTER_AN_OVERSEAS_ENTITY_WITH_PARAMS_URL + TRUSTS_URL;
 export const TRUST_INFO_WITH_PARAMS_URL = REGISTER_AN_OVERSEAS_ENTITY_WITH_PARAMS_URL + TRUST_INFO_PAGE;
+export const TRANSACTION_ID_URL_KEY = "/transaction/";
```

Comment:
Is this value (and one immediately below) really a 'key' or is it perhaps a sub-path within the URL?